src/chapters/booleans-and-conditionals/studio.rst

```diff
@@ -0,0 +1,59 @@
+Studio: Goal Setting and Getting into the Right Mindset
+=======================================================
+
+During this studio, we will ask you to think about your mindset when it comes to facing challenges.
+Do challenges deter you or delight you?
+If challenges are something you delight in, you might have a growth mindset.
+A **growth mindset** can be powered by one word, “yet”.
+The word “yet” allows us to acknowledge that we don’t know something AND that we are capable of learning it.
+For example, with “yet”, the sentence “I don’t understand booleans” becomes “I don’t understand booleans `yet`.”
+
+Growing as developers is something that will take our whole careers.
+No developer knows everything.
+As technology evolves, even the most senior developers have to learn new skills.
+Developing a framework for learning, which includes setting achievable goals, is important for being a developer.
+Technology will change and adapt and we should learn to do the same!
+
+When setting goals, those goals should be SMART. SMART stands for:
+
+**S** pecific
+
+**M** easurable
+
+**A** ttainable
+
+**R** elevant
+
+**T** ime Bound
+
+We will be discussing your goals when it comes to taking this class and your career, but you don't have to do any prior goal setting or preparation to make the most of this activity.
+
+.. note::
+
+   We will be making note of personal goals.
+   If there is a place where you like to write down your personal goals, such as a journal, feel free to bring it to class!
+
+Activity
+--------
+After your TA reviews the relevant topics, we will have a discussion covering the following:
+
+1. Your goals for taking this class. 
+	You probably have many reasons why this class is something that you want to complete.
+	Please share 1 to 2 of your goals and keep them SMART!
+
+2. Your goals for your career.
+	How does this class assist you in your career and where do you want to go?
+	Please share 1 to 2 goals for your career.
+
+3. Your action statements.
+	This can be anthing that motivates you from stories to quotes to ideas.
+	Please share 1 to 2.
+
+If there is any time at the end, make sure to get your goals and action statements written down somewhere where you will regularly see them!
+
+Resources
```

Comment:
How about adding a link to a "Fixed mindset" vs. "Growth mindset" graphic?  There are TONS of these online, but here is one nice, simple example. We might have to make our own, however.

![Fixed-Mindset-vs -Growth-Mindset-Chart](https://user-images.githubusercontent.com/31870126/58180038-ffb7e780-7c6e-11e9-864b-b3e7d9fc1844.jpg)